src/main/steps/common/common.content.ts

```diff
@@ -132,8 +138,14 @@ const cy: typeof en = {
   phase: 'Beta',
   applyForDivorce: 'Gwneud cais am ysgariad',
   applyForDissolution: 'gwneud cais i ddod â phartneriaeth sifil i ben',
-  feedback:
-    'Mae hwn yn wasanaeth newydd - <a class="govuk-link" aria-label="Dolen adborth, Bydd hyn yn agor tab newydd. Bydd angen ichi ddod yn ôl at y tab hwn a pharhau â’ch cais o fewn 60 munud fel na fyddwch yn colli’r gwaith yr ydych wedi ei wneud yn barod." href="https://www.smartsurvey.co.uk/s/Divorce_Feedback" target="_blank">bydd eich sylwadau</a> yn ein helpu i wella’r gwasanaeth.',
+  feedback: {
+    part1: 'Mae hwn yn wasanaeth newydd - ',
+    part2: 'bydd eich sylwadau',
+    part3: ' yn ein helpu i wella’r gwasanaeth.',
+    ariaLabel:
+      'Dolen adborth, Bydd hyn yn agor tab newydd. Bydd angen ichi ddod yn ôl at y tab hwn a pharhau â’ch cais o fewn 60 munud fel na fyddwch yn colli’r gwaith yr ydych wedi ei wneud yn barod.',
+    link: 'https://www.smartsurvey.co.uk/s/NFD_Feedback/?pageurl=',
```

Comment:
could we move this to config

Reply:
for some reason it fails when i do this? im not sure why